src/anndata/experimental/backed/_lazy_arrays.py

```diff
@@ -163,7 +163,7 @@ def dtype(self):
         elif self._dtype_str == "nullable-boolean":
             return pd.BooleanDtype()
         elif self._dtype_str == "nullable-string-array":
-            return pd.StringDtype()
+            return np.dtype("O")
```

Comment:
why? `pd.StringDtype` is better than object.

Reply:
Sorry, missed a spot where I should have linked the issue ttps://github.com/pydata/xarray/issues/10419